gen/dcos-config.yaml

```diff
@@ -1961,6 +1961,9 @@ package:
       LIBPROCESS_PORT=15201
       METRONOME_SCHEDULER_TASK_LAUNCH_CONFIRM_TIMEOUT=600000
       METRONOME_SCHEDULER_TASK_LAUNCH_TIMEOUT=600000
+      METRONOME_STATSD_HOST="localhost"
+      METRONOME_STATSD_PORT=61825
```

Comment:
This is the static port assigned for STATSD? Isn't 32000+ reserved for dynamic ports?

Reply:
jira:  says this is it:  https://jira.mesosphere.com/browse/DCOS_OSS-4935
same for marathon https://jira.mesosphere.com/browse/DCOS-47693
and what was used for the Marathon configuration:  https://github.com/dcos/dcos/pull/4323/files#diff-3f4e5f3f735bf0976a529aae710020bbR15

Double checking...

Reply:
yep...  https://github.com/dcos/dcos/blob/master/docs/admin-ports.md
```
UDP
53: dcos-net (dns)
61825: dcos-telegraf```

Reply:
Ah, Linux uses 32768-61000. So, this is not in the dynamic port range. I learned (or probably relearned) something. Everything looks good!